src/main/java/com/denizenscript/depenizen/bukkit/events/projectkorra/EntityBendingDeathScriptEvent.java

```diff
@@ -0,0 +1,79 @@
+package com.denizenscript.depenizen.bukkit.events.projectkorra;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.projectkorra.projectkorra.event.EntityBendingDeathEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class EntityBendingDeathScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // projectkorra player|entity dies|death|killed by <ability>
+    //
+    // @Triggers when an entity dies from bending.
+    //
+    // @Context
+    // <context.ability> returns ElementTag(String) of the ability's name.
```

Comment:
`(String)` is not a thing, and either way you don't need to specify types so heavily, just `<context.ability> returns the ability's name.`